main/src/ca/uwaterloo/flix/language/ast/Ast.scala

```diff
@@ -789,7 +789,7 @@ object Ast {
 
       case object Do extends Expr
 
-      case class InvokeMethod(e: TypeError.MethodNotFound) extends Expr
+      case class InvokeMethod(tpe: ca.uwaterloo.flix.language.ast.Type, name: Name.Ident) extends Expr
```

Comment:
`TypeContext.Type` steals the scope - I couldn't find a way to shorthand the import

Reply:
I suppose that is OK

Reply:
Do we actually need this syntactic context? Are the errors not enough by themselves?